api_schema/schema.go

```diff
@@ -524,6 +524,17 @@ type PostClientPacketCaptureStatsRequest struct {
 	// Report on any errors encounted during apidump that should be shown to the user.
 	ApidumpError     ApidumpErrorType `json:"apidump_error,omitempty"`
 	ApidumpErrorText string           `json:"apidump_error_text,omitempty"`
+
+	// Report CPU and memory usage, if available.
+	AgentUsage *AgentUsage `json:"agent_usage,omitempty"`
+}
+
+type AgentUsage struct {
+	// Akita agent CPU usage, as a percentage of total CPU availability.
+	RelativeCPU float64 `json:"relative_cpu"`
+
+	// Peak virtual memory size in bytes.
+	VMPeak uint64 `json:"vm_peak"`
```

Comment:
This should be peak RSS.

Reply:
Fixed in 9a50cbd4827ff573e2d59bc69a8cfe7f73971278.